src/dag/verify.js

```diff
@@ -53,7 +53,10 @@ export default function(root) {
         .forEach((span) => span.forEach((n) => (otherSpan[n] = true)));
       return rootSpan.some((n) => otherSpan[n]);
     });
-  if (!connected) throw new Error("dag was not connected");
+  if (!connected) {
+    // eslint-disable-next-line no-console
+    console.warn("dag was not connected");
+  }
```

Comment:
I don't actually think this is necessary. In general javascript fashion, if this "works" then let it happen. I might instead opt (and you don't have to include this as part of the diff, but it would be nice) for a connected method that just returns true if that dag is connected.